src/client/Microsoft.Identity.Client/Platforms/iOS/Broker/iOSBroker.cs

```diff
@@ -51,15 +51,10 @@ public bool IsBrokerInstalledAndInvokable()
         {
             using (_logger.LogMethodDuration())
             {
-                if (_uIParent?.CallerViewController == null)
-                {
-                    _logger.Error(iOSBrokerConstants.CallerViewControllerIsNullCannotInvokeBroker);
-                    throw new MsalClientException(MsalError.UIViewControllerRequiredForiOSBroker, MsalErrorMessage.UIViewControllerIsRequiredToInvokeiOSBroker);
-                }
-
+                NSObject obj = new NSObject();
```

Comment:
can you just invoke this if the CallerViewController is null? 

also, what is the e2e with this and the silent flow?